### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by dangrous**
$ https://github.com/Expensify/Expensify/issues/334627

This updates the design to match @dubielzyk-expensify's requests.

@tgolen I couldn't reproduce the [resetting of the filters](https://expensify.slack.com/archives/C03TQ48KC/p1699459489261079). Might that have been fixed with your recent 1.3.6 update? It looks like these remain static even when the new data is fetched.

### CODE REVIEW

**Comment by tgolen** [comment]
Yeah, I agree. I haven't been able to reproduce the problem with the filters again either.

File: src/css/content.scss [hunk 1]
```diff
@@ -559,5 +560,16 @@ $color-dark-yellow: #DAA520;
...
 }
+
+@media (prefers-color-scheme: dark) {
+  .panel, .panel-item, .issue-filter {
+     border-color: rgb(48, 54, 61);
+  }
+  .issue-filter {
+     color: rgb(230, 237, 243)
+  }
+}
```
Comment (thienlnam) [inline src/css/content.scss:575]: Looks like we are missing a new line here

**Review by thienlnam — CHANGES_REQUESTED** [review]
Looking good, just one small comment

**Comment by dangrous** [comment]
updated!

**Review by thienlnam — APPROVED** [review]

**Pull request merged**